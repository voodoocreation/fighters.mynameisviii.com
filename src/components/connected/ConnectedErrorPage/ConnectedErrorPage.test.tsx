import ComponentTester from "../../../utilities/ComponentTester";

import ConnectedErrorPage from "./ConnectedErrorPage";

const component = new ComponentTester(
  ConnectedErrorPage,
  true
).withDefaultReduxState({
  page: {
    error: {
      message: "Not found",
      status: 404
    }
  }
});

describe("[connected] <ConnectedErrorPage />", () => {
  it("renders correctly", () => {
    const { actual } = component.render();

    expect(actual).toMatchSnapshot();
  });
});
