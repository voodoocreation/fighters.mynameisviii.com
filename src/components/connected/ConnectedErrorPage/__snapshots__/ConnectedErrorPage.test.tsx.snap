// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[connected] <ConnectedErrorPage /> renders correctly 1`] = `
<article
  class="ErrorPage"
>
  <h1>
    Page not found
  </h1>
  <div
    class="ErrorPage--content container"
  >
    <p>
      This page does not exist.
    </p>
  </div>
</article>
`;
